mirage/index.js

```diff
@@ -2,8 +2,14 @@
 
 let start = () => {};
 
-if (process.env.NODE_ENV !== 'production') {
-  start = require('./start').default;
-}
+// currently we run mirage in production
+// when we switch over to the initial api protect this
+// require with an if statement so that webpack won't
+// attempt to bundle mirage.
```

Comment:
i'd add this punctuation:
// currently we run mirage in production`.`
// when we switch over to the initial api`,` protect this
// require with an if statement so that webpack won't
// attempt to bundle mirage.